src/Parlot/Fluent/Error.cs

```diff
@@ -8,28 +8,15 @@
 
 namespace Parlot.Fluent;
 
-public sealed class ElseError<T> : Parser<T>, ICompilable, ISeekable
```

Comment:
This is a breaking change

Reply:
This is a marker class, doesn't matter.

Reply:
While it's a public API it's a breaking-change :)